lib/collection/src/optimizers_builder.rs

```diff
@@ -77,6 +79,15 @@ pub struct OptimizersConfig {
     /// If 0 - no optimization threads, optimizations will be disabled.
     #[serde(default)]
     pub max_optimization_threads: Option<usize>,
+
+    /// Payload keys to use for defragmentation. The keys configured will be used
+    /// to sort vectors internally to take advantage from memory/storage locality.
+    /// The keys are applied in order, where the first key is most important.
+    /// (For now, only the firts one is applied)
+    ///
+    /// Note: The keys need to be indexed.
```

Comment:
This may be a nice nuance:

```suggestion
    /// Note: The keys need to be indexed with a type that supports ordering.
```